<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmVideoPlayWeb</class>
 <widget class="QWidget" name="frmVideoPlayWeb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetVideo" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="widgetRight" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frameRight">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="3" column="1">
          <widget class="QComboBox" name="cboxRtsp"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labDateTimeStart">
           <property name="text">
            <string>开始时间</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDateTimeEdit" name="dateTimeStart">
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cboxType"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labType">
           <property name="text">
            <string>视频类型</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cboxCh">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labCh">
           <property name="text">
            <string>设备通道</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labSpeed">
           <property name="text">
            <string>播放速度</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="cboxSpeed"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labDateTimeEnd">
           <property name="text">
            <string>结束时间</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labRtsp">
           <property name="text">
            <string>码流地址</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEnd">
           <property name="displayFormat">
            <string>yyyy-MM-dd HH:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnDelete">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnDownload">
             <property name="text">
              <string>下载</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnClear">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btnSelect">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btnSelectAll">
             <property name="text">
              <string>全选</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btnSelectNone">
             <property name="text">
              <string>不选</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labNvr">
           <property name="text">
            <string>设备地址</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cboxNvr"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="VideoPlayback" name="videoPlayback" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoPlayback</class>
   <extends>QWidget</extends>
   <header>videoplayback.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cboxNvr</tabstop>
  <tabstop>cboxCh</tabstop>
  <tabstop>cboxType</tabstop>
  <tabstop>cboxRtsp</tabstop>
  <tabstop>cboxSpeed</tabstop>
  <tabstop>dateTimeStart</tabstop>
  <tabstop>dateTimeEnd</tabstop>
  <tabstop>btnSelect</tabstop>
  <tabstop>btnDownload</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>btnSelectAll</tabstop>
  <tabstop>btnSelectNone</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
